packages/react-composites/tests/app/call/BaseApp.tsx

```diff
@@ -38,9 +38,26 @@ export function BaseApp(props: { queryArgs: QueryArgs; callAdapter?: CallAdapter
     ? () => alert('you are using a unsupported browser')
     : undefined;
 
+  const onPermissionsTroubleshootingClick = queryArgs.useCallReadiness
+    ? (permissionsState: { camera: PermissionState; microphone: PermissionState }) => {
+        alert(permissionsState);
+      }
+    : undefined;
+
   let customCallCompositeOptions;
 
-  if (onEnvironmentInfoTroubleshootingClick) {
+  if (onEnvironmentInfoTroubleshootingClick && onPermissionsTroubleshootingClick) {
```

Comment:
Flatten this to two if statements rather than the complex three-way if-elseif.

In fact, better to merge the conditional above with this one.

```
let customCallCompositeOptions = queryArgs.customCallCompositeOptions

if (queryArgs.onEnvironmentInfoTroubleshootingActions) {
  customCallCompositeOptions = {
    ...customCallCompositeOptions,
    onEnvironmentInfoTroubleshootingClick: onEnvironmentInfoTroubleshootingClick,
  }
}

if (queryArgs.onPermissionsTroubleshootingOptions) {
  customCallCompositeOptions = {
    ...customCallCompositeOptions,
    onPermissionsTroubleshootingClick: onPermissionsTroubleshootingClick,
  }
}
```

where `onEnvironmentInfoTroubleshootingClick` and `onPermissionsTroubleshootingClick` are free functions with the relevant implementation.